index.html

```diff
@@ -0,0 +1,235 @@
+          </div>
+          </div>
+          <div class="card">
+              <img src="images/multiple2.png" alt="multiple2"> 
+              <div class="desc">
+           <h2>Multi-post stories</h2>
+           <div class="tonic-items-title">
+            <p class="item1" >CANOPY</p>
+           <div class="list-tonic">
+          <ul class="tonic-items">
+            <li class="dot"></li>
+           <li class="tonic-item2">Back End Dev</li>
+           <li class="dot"></li>
+           <li class="tonic-item3">2015</li>
+         </ul>
+       </div>
+       </div>
+          <p class="tonic-description">A daily selection of privately personalized reads; no accounts or
+            sign-ups required.</p>
+          <ul class="capabilities">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="button" class="btn-tonic">See Project</button>
+          </div>
+          </div>
+          </section>
+            <!--ABOUT MYSELF SECTION-->
+          <section class="about">
+              <div class="Description">
+            <h1 class="heading">About <br/> myself</h1>
+            <p class="profile">
+              I’m a software developer! I can help you build a product , feature or
+              website Look through some of my work and experience! If you like what
+              you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <p class="info">LET'S CONNECT</p>
+            <div class="media-icon">
+                <ul>
+                    <li class="icon">
+                        <a href="#">
+                          <img src="images/twitter.png" alt="Twitter">
+                        </a>
+                    </li>
+                    <li class="icon">
+                        <a href="#">
+                          <img src="images/Linkedin.png" alt="LinkedIn">
+                        </a>
+                    </li>
+                    <li class="icon">
+                        <a href="#">
+                          <img src="images/media.png" alt="media">
+                        </a>
+                    </li>
+                    <li class="icon">
+                        <a href="#">
+                            <img src="images/github.png" alt="Github"></a>
+                    </li>
+                    <li class="icon">
+                        <a href="#">
+                          <img src="images/fingers.png" alt="Finger">
+                        </a>
+                    </li>
+                </ul>
+            </div>
+              <input type="button" class="btn-tonic" value="Get my resume">
+            </div>
+
+            <div class="language-icon">
+            <div class="language-title">
+            <p>Language</p>
+            <i class="fas fa-chevron-right"></i>
+            </div>
+            <div class="language">
+              <ul class="language-list">
```

Comment:
- [ ] Here the design should same as the original website.
- your website
![edit1](https://user-images.githubusercontent.com/26767383/149563604-2446bc36-16b4-4c62-8fa6-e84c4afd8df8.PNG)
- Original website
![original1](https://user-images.githubusercontent.com/26767383/149563639-d9ab602e-f0fd-4a70-b4a3-27b1d8a0bcc0.PNG)